tvb_framework/tvb/config/logger/dev_logger_config.conf

```diff
@@ -82,6 +82,12 @@ handlers=consoleHandler, fileHandler
 qualname=matplotlib
 propagate=0
 
+[logger_elastic_transport]
+level=ERROR
+handlers=consoleHandler, fileHandler
+qualname=elastic_transport
+propagate=0
+
```

Comment:
What is the purpose of this? to differentiate logs from elastic ?

Reply:
To suppress logs specific to sending the logging information (from the elasticserach logger) to elasticsearch.